cosmwasm/deployment/terra2/tools/deploy.js

```diff
@@ -15,14 +15,13 @@ import { zeroPad } from "ethers/lib/utils.js";
   deterministic.
 */
 const artifacts = [
-  "wormhole.wasm",
+  "wormhole_ibc.wasm",
```

Comment:
did the core contract get renamed from wormhole to wormhole_ibc? if so, why? if not, don't we need to deploy the actual core contract?

Reply:
No, see the IBC cosmwasm contracts PR: https://github.com/wormhole-foundation/wormhole/pull/2591

`wormole_ibc` is a new contract that we will be deploying to IBC enabled chains such as terra2. It is meant to replace the core bridge contract. For the devnet deployment, we can just replace the contract entirely. For the production migration on existing Cosmos deployments, we will first upload the `wormhole-ibc` code, and then migrate the `wormole` contract to use the `wormhole-ibc` code.

Reply:
Could the IBC version be deployed _in addition to_ the existing core wormhole contract. Then the IBC specific testing could be done alongside the existing tests. Making this strictly additional and not a replacement.